__tests__/interceptors/checkReRun.test.js

```diff
@@ -3,34 +3,58 @@ const CheckReRun = require('../../lib/interceptors/checkReRun')
 const Helper = require('../../__fixtures__/helper')
 require('object-dot').extend()
 
-test('#possibleInjection', () => {
-  let checkReRun = new CheckReRun()
-
-  expect(
-    checkReRun.possibleInjection(Helper.mockContext(), {id: 1}, {id: 1})
-  ).toBe(false)
-  expect(
-    checkReRun.possibleInjection(Helper.mockContext(), {id: 1}, {id: 2})
-  ).toBe(true)
-})
-
-test('#process', async () => {
+test('return if pre conditions are not met', async () => {
   let checkReRun = new CheckReRun()
   let context = Helper.mockContext()
 
   context.event = 'check_run'
-  context.payload.action = 'rerequested'
+  context.payload.action = 'opened'
   Object.set(context, 'payload.check_run.output.text', mockOutput())
-  context.payload.check_run.pull_requests = [{number: 1}]
-  context.payload.check_run.id = 123
   context.github.pullRequests.get.mockReturnValue({ data: { number: 456 } })
   let newContext = await checkReRun.process(context)
 
-  expect(newContext.payload.pull_request.number).toBe(456)
+  expect(newContext.event).toBe('check_run')
+
```

Comment:
I would very much prefer for the tests to be kept separate:

1. Where `#process` fully executes (as per the original test) 
2. Where the event and action conditions are not met. The inverse of the above and the bug you found.

It would be more readable. Right now this test is checking for both scenarios while the title says "'return if pre conditions are not met'" -- which is rather confusing.

Do you mind keeping the original (commented out) and changing this test to only test for when the conditions are not met as your title implies?

Reply:
done